.github/actions/publish-logs/action.yml

```diff
@@ -17,6 +17,7 @@ runs:
         ./eden metric --format json > metric.log || echo "no metric"
         ./eden netstat --format json > netstat.log || echo "no netstat"
         cp dist/default-eve.log console.log || echo "no device log"
+        cp dist/qmp.log qmp.log || echo "no qmp log"
```

Comment:
Sorry, seems I come too late. Small note here: we have `default-` prefix for reason (above). It is config context (we can create another with non-default name) we may use to run several EVE-OS under Eden control (I know, it is not easy because of port's conflict, but at least possible with some crafted config). With two qemu expected to run on the same qmp.sock it become impossible. So it would be great to not use dist/qmp.log and dist/qmp.sock, but use ones prefixed by context name.

Reply:
This is easy to fix, I can do this. I expected this kind of thing, and my initial thoughts to have "default-" prefix was correct, but I had another problem with the path, so forgot. Thanks for bringing that back.

If it won't be difficult for you, could you please take a look into my problem with the `currentPath` and `os.Getwd()`, described here: https://github.com/lf-edge/eden/pull/1004#issuecomment-2247917847

Reply:
Here https://github.com/lf-edge/eden/pull/1007

Reply:
Well, inside of tests we use temp directory as a current to keep all files and artifacts there. So you cannot use current directory there to point onto dist files. You can try to use `utils.ResolveAbsPath` instead which will load use stored path to dict directory in config and append the argument to it.